src/core/Pagination/Pagination.md

```diff
@@ -228,6 +248,108 @@ const fakePageNode = (
     aria-label="Pagination"
     onChange={(page) => {
       setCurrent(page);
+      headingRef.current.focus();
+    }}
+    pageIndicatorText={(currentPage, lastPage) =>
+      'Page ' + currentPage + ' / ' + lastPage
+    }
+    ariaPageIndicatorText={(currentPage, lastPage) =>
+      'Showing page ' + currentPage + ' out of ' + lastPage
+    }
+  />
+</Block>;
+```
+
+### Page browsing using links
+
+For search engine optimization or other such reasons, it is possible to give custom components to use instead of the regular arrow buttons for browsing. If you do decide to replace the buttons, make sure that the customized solution is accessible. When the first or last page is reach via clicking the links or other custom components, move focus to the other browse button.
+
+```js
+import {
+  Pagination,
+  Block,
+  Heading,
+  Link,
+  Text,
+  IconArrowRight,
+  IconArrowLeft,
+  suomifiDesignTokens
+} from 'suomifi-ui-components';
+import { useRef } from 'react';
+
+const [current, setCurrent] = React.useState(2);
+const lastPage = 8;
+const prevPageLinkRef = useRef();
+const nextPageLinkRef = useRef();
+const headingRef = useRef();
+
+const fakePageNode = (
+  <Block
+    padding="xl"
+    mb="l"
+    style={{
+      border: `1px solid ${suomifiDesignTokens.colors.depthLight1}`
+    }}
+  >
+    <Heading variant="h3" forwardedRef={headingRef} tabIndex="-1">
+      Page: {current}
+    </Heading>
+  </Block>
+);
+
+const handlePageChange = (newPage) => {
+  if (newPage <= 0 || newPage > lastPage) return;
+  setCurrent(newPage);
+  headingRef.current.focus();
+};
+
+<Block style={{ width: '600px' }}>
+  {fakePageNode}
+  <Pagination
+    currentPage={current}
+    lastPage={lastPage}
+    customPreviousButton={
+      current <= 1 ? (
+        <Text style={{ color: suomifiDesignTokens.colors.depthBase }}>
+          Previous page
+        </Text>
+      ) : (
+        <Link
+          forwardedRef={prevPageLinkRef}
+          href="./#/Components/Pagination?id=page-browsing-using-links"
+          rel="prev"
+          onClick={() => handlePageChange(current - 1)}
+        >
+          Previous page
+        </Link>
+      )
+    }
+    customNextButton={
+      current >= lastPage ? (
+        <Text style={{ color: suomifiDesignTokens.colors.depthBase }}>
+          Next page
+        </Text>
+      ) : (
+        <Link
+          forwardedRef={nextPageLinkRef}
+          href="./#/Components/Pagination?id=page-browsing-using-links"
+          rel="next"
+          onClick={() => handlePageChange(current + 1)}
+        >
+          Next page
+        </Link>
+      )
+    }
+    aria-label="Pagination"
+    pageInputProps={{
+      invalidValueErrorText: (value) => `"${value}" is not allowed`,
+      inputPlaceholderText: 'Go to',
+      buttonText: 'Jump to page',
+      labelText: 'Page number'
+    }}
```

Comment:
These are redundant in this case, no pageinput in use

Reply:
This should now be cleaned 👍